src/mozilla_taskgraph/transforms/scriptworker/shipit/mark_as_shipped.py

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+
+from taskgraph.transforms.base import TransformSequence
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def make_task_description(config, tasks):
+    if "shipit" not in config.graph_config:
+        config_path = os.path.join(config.graph_config.root_dir, "config.yml")
+        raise Exception(f"Missing 'shipit' config in {config_path}")
+
+    shipit = config.graph_config["shipit"]
+    release_format = shipit.get(
+        "release-format", "{product}-{version}-build{build_number}"
+    )
+    scope_prefix = shipit.get("scope-prefix", "project:releng:ship-it")
+    shipit_server = "production" if config.params["level"] == "3" else "staging"
+    version = config.params.get("version", "<ver>")
```

Comment:
I seem to remember: `<ver>` is just a magic string that has no effect whatsoever, right?

Reply:
Yeah, that's what `firefox-android` had so I just copied it here. Maybe we could do something better? I think that could be a good follow-up